File: src/main/java/blog/dao/ManagerDao.java
package blog.dao;

import org.apache.ibatis.annotations.Param;
import org.springframework.stereotype.Repository;

import java.util.Date;


@Repository("blog/dao/ManagerDao")
public interface ManagerDao {

    int insertBlog(@Param("title") String title, @Param("permission") int permission, @Param("summary") String summary,
                   @Param("writer") String writer, @Param("backgroundImage") String backgroundImage, @Param("date") Date date,
                   @Param("typeId") int typeId, @Param("allowComment") int allowComment, @Param("wordTextId") String wordTextId);

    int insertWordText(@Param("text") String text, @Param("wordTextId") String wordTextId);

}

File: src/main/kotlin/blog/dao/ManagerDao.kt
package blog.dao

import org.apache.ibatis.annotations.Mapper
import org.apache.ibatis.annotations.Param
import java.util.*

/**
 * @author Frank
 * @date 2018/7/7 15:50
 */
@Mapper
interface ManagerDao {

    fun insertBlog(@Param("title") title: String, @Param("permission") permission: Int, @Param("summary") summary: String,
                   @Param("writer") writer: String, @Param("backgroundImage") backgroundImage: String,
                   @Param("date") date: Date, @Param("typeId") typeId: Int, @Param("allowComment") allowComment: Int,
                   @Param("wordTextId") wordTextId: String): Int

    fun insertWordText(@Param("text") text: String, @Param("wordTextId") wordTextId: String): Int
}